_benchmark/quickstart.md

```diff
@@ -0,0 +1,421 @@
+|                                       100th percentile latency | desc_sort_with_after_geonameid |     24.3476 |      ms |
+|                                  100th percentile service time | desc_sort_with_after_geonameid |     6.81395 |      ms |
+|                                                     error rate | desc_sort_with_after_geonameid |           0 |       % |
+|                                                 Min Throughput |             asc_sort_geonameid |       69.44 |   ops/s |
+|                                                Mean Throughput |             asc_sort_geonameid |       69.44 |   ops/s |
+|                                              Median Throughput |             asc_sort_geonameid |       69.44 |   ops/s |
+|                                                 Max Throughput |             asc_sort_geonameid |       69.44 |   ops/s |
+|                                       100th percentile latency |             asc_sort_geonameid |     19.4046 |      ms |
+|                                  100th percentile service time |             asc_sort_geonameid |     4.72967 |      ms |
+|                                                     error rate |             asc_sort_geonameid |           0 |       % |
+|                                                 Min Throughput |  asc_sort_with_after_geonameid |       70.35 |   ops/s |
+|                                                Mean Throughput |  asc_sort_with_after_geonameid |       70.35 |   ops/s |
+|                                              Median Throughput |  asc_sort_with_after_geonameid |       70.35 |   ops/s |
+|                                                 Max Throughput |  asc_sort_with_after_geonameid |       70.35 |   ops/s |
+|                                       100th percentile latency |  asc_sort_with_after_geonameid |      18.664 |      ms |
+|                                  100th percentile service time |  asc_sort_with_after_geonameid |     4.16119 |      ms |
+|                                                     error rate |  asc_sort_with_after_geonameid |           0 |       % |
+
+
+--------------------------------
+[INFO] SUCCESS (took 98 seconds)
+--------------------------------
+```
+
+Each task ran by the `geonames` workload represents a specific OpenSearch API operation, such as Bulk or Search, that was performed when the test was ran. Each task in the output summary contains the the following information: 
+
+* **Throughput:** The number of successful OpenSearch operations per second. 
+* **Latency:** The time, including wait time, it took for the request and the response to be sent out and received by OSB
+* **Service Time:** The time, excluding wait time, it took for the request and the response to be sent out and received by OSB
+* **Error Rate:** Percent of operations run during the task that were not successful or (200s)
+
```

Comment:
We will most likely have to revise Latency and Service Time as these have different meanings in OSB that we don't want people to confused about. When we publish the concept page over Throughput, Service Time, and Latency, we can revise these definitions and also include a sentence saying "For more insight into Throughput, Service Time, and Latency see this discussion"